test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,23 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+    }
+
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
I think you shouldn't name the method `isAnyNull` since there already exists a method with the same name in [here](https://github.com/se-edu/addressbook-level2/blob/master/src/seedu/addressbook/common/Utils.java#L15).

Reply:
Also, one more test case you can try is empty `Objects` parameter.